woocommerce-myparcel.php

```diff
@@ -376,15 +379,24 @@ public function plugin_url()
          *
          * @return string
          */
-        public function plugin_path()
+        public function plugin_path(): string
         {
             return untrailingslashit(plugin_dir_path(__FILE__));
         }
 
+        /**
+         * @return void
+         * @throws \Exception
+         */
+        public function initDashboardWidget(): void
+        {
+            (new MyParcelWidget())->loadWidget();
```

Comment:
dit is niet nodig, je had `[new MyParcelWidget(), 'loadWidget']` als callback kunnen doen